### ISSUE TITLE
Is this a feature, or a bug?

### ISSUE THREAD

**Issue by agucova** [stonesam92/ChitChat#119]
The gap in the menu bar... It's not common in most apps...
![Uploading overlap2.png…]()

**Comment by agucova** [comment]
The image didn't upload... 
<img width="1044" alt="overlap2" src="https://cloud.githubusercontent.com/assets/4694408/12072535/817351a6-b0c1-11e5-9631-038b0071fe24.png">

**Comment by alexpearce** [comment]
How do you mean “gap”? Do you mean that the grey bar in the top left, containing the window controls, doesn't extend all the way to the far right? That's intentional. The space to the right is filled with a conversation information bar when you click on a conversation in the list on the right.

**Comment by iDuuck** [comment]
What screen is this? Normally the menubar is going through to the very right corner.

**Comment by alexpearce** [comment]
Do you mean “normally” as in ‘normally in ChitChat’ or ‘normally in OS X applications’?

**Comment by iDuuck** [comment]
Normally in WhatsApp for Web, so also ChitChat.

**Comment by agucova** [comment]
No I mean, vertically, the buttons should be right under the border.

**Comment by agucova** [comment]
This is a 13 inch display.

**Comment by reitermarkus** [comment]
You mean this, right? 
![buttons-top](https://cloud.githubusercontent.com/assets/1309829/12296798/2a70673e-ba0a-11e5-83f3-d1f94925fe74.png) ![buttons-middle](https://cloud.githubusercontent.com/assets/1309829/12296797/2a49caf2-ba0a-11e5-8468-199de9881671.png)

**Comment by iDuuck** [comment]
It’s normal that the buttons are centered on the titlebar. That’s how some applications are doing it. And if the title bar is that high, it’s better from design aspect.

**Comment by reitermarkus** [comment]
Yeah, iTunes is a good example since it has a higher-than-usual titlebar.

**Comment by iDuuck** [comment]
Chrome, and many apple apps are doing this, yeah. So feature & not a bug.
